src/pages/settings/Payments/PaymentsPage/paymentsPagePropTypes.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import walletTransferPropTypes from '../walletTransferPropTypes';
+import {withLocalizePropTypes} from '../../../../components/withLocalize';
+import {windowDimensionsPropTypes} from '../../../../components/withWindowDimensions';
+
+const propTypes = {
+    /** Wallet balance transfer props */
+    walletTransfer: walletTransferPropTypes,
+
+    /** List of betas available to current user */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
+    /** Are we loading payment methods? */
+    isLoadingPaymentMethods: PropTypes.bool,
+
+    /** Is resize event happens on the web/desktop? */
```

Comment:
Not a blocker: Change the comment to "Listen for window resize event on web and desktop."